CHANGELOG.md

```diff
@@ -7,6 +7,45 @@
 
 # Release notes
 
+## 1.0.0
+
```

Comment:
I think we need a paragraph here about what it means to switch from xwalk to webview, in particular...

- Much newer version of Chrome which means better performance, fewer bugs, better security, and more features.
- Much smaller apk size (document the difference here)
- Dropping xwalk library which is years out of date and completely unsupported
- Support for upcoming versions of cht-core.
- Anything else?